OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -230,6 +231,7 @@
     <Compile Include="Scripting\Properties\GuardProperties.cs" />
     <Compile Include="Scripting\Properties\MissionObjectiveProperties.cs" />
     <Compile Include="Scripting\Properties\MobileProperties.cs" />
+    <Compile Include="Scripting\Properties\DemolitionProperties.cs" />
```

Comment:
Then again just put this into a separate pull request so we can merge it later with a test case.